pkg/config/validation/extensionprovider.go

```diff
@@ -87,6 +99,48 @@ func validateExtensionProviderEnvoyExtAuthzGRPC(config *meshconfig.MeshConfig_Ex
 	return
 }
 
+func validateExtensionProviderTracingZipkin(config *meshconfig.MeshConfig_ExtensionProvider_ZipkinTracingProvider) (errs error) {
+	if config == nil {
+		return fmt.Errorf("nil TracingZipkinProvider")
+	}
+	if err := validateExtensionProviderService(config.Service); err != nil {
+		errs = appendErrors(errs, err)
+	}
+	if err := ValidatePort(int(config.Port)); err != nil {
+		errs = appendErrors(errs, fmt.Errorf("invalid service port: %v", err))
+	}
+	return
+}
+
+func validateExtensionProviderTracingLightStep(config *meshconfig.MeshConfig_ExtensionProvider_LightstepTracingProvider) (errs error) {
+	if config == nil {
+		return fmt.Errorf("nil TracingLightStepProvider")
+	}
+	if err := validateExtensionProviderService(config.Service); err != nil {
+		errs = appendErrors(errs, err)
+	}
+	if err := ValidatePort(int(config.Port)); err != nil {
+		errs = appendErrors(errs, fmt.Errorf("invalid service port: %v", err))
+	}
+	if config.AccessToken == "" {
+		errs = appendErrors(errs, fmt.Errorf("access token is required"))
+	}
+	return
+}
+
+func validateExtensionProviderTracingDatadog(config *meshconfig.MeshConfig_ExtensionProvider_DatadogTracingProvider) (errs error) {
+	if config == nil {
+		return fmt.Errorf("nil TracingDatadogProvider")
+	}
+	if err := validateExtensionProviderService(config.Service); err != nil {
+		errs = appendErrors(errs, err)
+	}
+	if err := ValidatePort(int(config.Port)); err != nil {
+		errs = appendErrors(errs, fmt.Errorf("invalid service port: %v", err))
+	}
+	return
+}
+
```

Comment:
do we want to add validation for OpenCensus too?

Reply:
added